### PR TITLE
Feature: created buttons component create buttons and OR divider

### PR DESCRIPTION
**Pull request by SufyanCS**
Created authentication Buttons with OR Divider component and make it responsive for Desktop, Tablet and Mobile.
Components struture and Desktop version:
ButtonAuth-Desktop
Tablet and Mobile version:
ButtonAuth-Tablet-Mobile

### CODE REVIEW

File: src/components/ButtonAuth.module.css
Comment (adeeb-samad) [inline src/components/ButtonAuth.module.css:1]: I preferred using tailwind

File: src/components/SignOptions.jsx
Comment (adeeb-samad) [inline src/components/SignOptions.jsx:1]: instead of ../../ you can use ~/

**Review by adeeb-samad — APPROVED** [review]
Done

**Review by sulmi24 — APPROVED** [review]
Thank you for submitting the pull request and for your work on this project. I have reviewed the changes that you have made. Overall, I think that the changes are well-implemented and consistent with the overall codebase.

File: src/components/SignOptions.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import ButtonAuth from "./ButtonAuth";
+import OrDivider from "./OrDivider";
+import GoogleIcon from "../../public/GoogleIcon.png";
```
Comment (iiAbady) [inline src/components/SignOptions.jsx:3]: Good work! As @Adeeb2022 said, using the `~/` is preferable to using something like `../../`.

**Review by iiAbady — APPROVED** [review]

**Pull request merged**